### DISCUSSION TITLE
Autolink References Not Valid When Nesting <num> in the Hostname

### DISCUSSION THREAD

**Discussion by tlambrou** [community/community#23420] [New to GitHub]
<p>I am trying to create an autolink reference to Netlify’s deploy preview url format.  Netlify uses the following format for determining URLs to their deploy previews which build for every Github PR.</p>
<p>https://deploy-preview-<code>[PR number]</code>–projectname.netlify.app</p>
<p>However when I try to put <code>&lt;num&gt;</code> in the in the url I get a <code>Target URL is invalid</code>.  This is a common format for a popular service that would be useful for autolink references to be working for.</p>
<p>You should allow <code>&lt;num&gt;</code> to be added to the hostname instead of strictly as a param in the url.</p>
<p>Here is a screenshot…<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/5/8/58b69c01785b93c7cf482087880d63eabed94ab4.jpeg" data-download-href="/uploads/short-url/cENfSWzTwxQWRyH5xoLFz9391ru.jpeg?dl=1" title="Add_autolink_reference" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/8610506/181076851-418923c3-2ace-47f2-8991-a1a58eab7a65.jpeg" alt="Add_autolink_reference" data-base62-sha1="cENfSWzTwxQWRyH5xoLFz9391ru" width="509" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/5/8/58b69c01785b93c7cf482087880d63eabed94ab4_2_509x500.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/5/8/58b69c01785b93c7cf482087880d63eabed94ab4_2_763x750.jpeg 1.5x, https://aws1.discourse-cdn.com/github/original/3X/5/8/58b69c01785b93c7cf482087880d63eabed94ab4.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/5/8/58b69c01785b93c7cf482087880d63eabed94ab4_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Add_autolink_reference</span><span class="informations">899×883 94.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by vgrl** [comment]
<p>Hi <a class="mention" href="/tlambrou">@tlambrou</a>—thanks for taking the time to reach out in our GitHub community. We appreciate your detailed feedback, and I’ve taken note of your suggestion and screenshot (thank you!) and passed it onto the appropriate engineering team for consideration in future conversations about how to improve this feature and GitHub.</p>
<p>In the future, the best way to provide feedback is to submit this through our <a href="https://support.github.com/contact/feedback">product feedback form</a>—which gets logged with our product team. While we cannot respond to all feedback submissions, we do consider every request and really appreciate our users taking the time to give us feedback.</p>
<p>Thanks again for sharing this feedback with us.</p>

**Reply by mbonaci** [reply]
The `product feedback form` leads to discussions, where this issue is now shown too, it's just missing the "Product feedback" label and you should unmark your comment as the answer, so that it actually gets seen by whomever is responsible for triage.

**Reply by tlambrou** [reply]
Hi @vgrl thanks for the info.  As @mbonaci pointed out, the product feedback form you linked just goes directly to [this link](https://github.com/orgs/community/discussions). Do you perhaps have another working link so we can properly submit this feedback?

**Comment by mbonaci** [comment]
Isn't this the whole point of this feature?  
I wanted to create autolinks with this type of URL: `https://pr-<num>.apps.test.company.com` which still doesn't seem to be possible, 2 years after the issue was reported.
Any news on this @vgrl?
